README.md

```diff
@@ -1,28 +1,66 @@
 
 ## **What problem does it solve?**
-    Overcome the problem of navigation in large stores. 
+- Overcome the problem of navigation in large stores. 
 
 ## **How are we going to make it?**
-App/webpage: One interface for the customer - one interface for the shop owner/employees. Customers: are able to use voice commands to ask the car
-(assistant) about items they are searching for and the car will tell the customer where the item is located; possibly show a path via an interface to the
-desired item, based on the customer's current location, which the customer could follow in order to find the item. Shop owners/employees: are going to be
-able to control the car manually or let the car drive autonomously in the store to relocate or to navigate to a certain location. The car (assistant):
-would roam around in the store aimlessly and will have an obstacle avoidance feature so it doesn't get stuck.
-## **What kind of technology we are going to use?**
+- We have done this by creating a website which will be running on the display of the shopping assistant while it navigates through a premise. The users can interact with the display and with speech functionality the user can ask the assistant questions and be given a voiced or visual response based on the question. The need to cover a large area and not be stationary enables the assistant to interact with more users. Since we want the need of personal and shopping assistant interactions to be minimized, the assistant is equipped with different kinds of self-navigation modes.
+
+## **What kind of technology have we used?**
     
     C++, HTML, CSS
     Web server 
     Voice recognition
     Arduino
-    Map development
-    GUI development 
-    Car logic development
-    Manhattan Path
+    Mqtt broker
+
+## **System diagram**
+![Assistant Diagram](https://user-images.githubusercontent.com/90680904/170860029-c5408425-946f-41ca-8c2a-7f29d9ea52e5.png)
+
+
+### **Diagram Discription**
+The system consists of three parts, the ardurino car, mqtt broker and the website. The shopping assistant and the website communicate via the mqtt broker. This enables the the user to controll the car from the website by publishing topics to the broker.  
+   
+## **Getting started**
+
+**1.**   Clone the repository.
+Please observe that step 1.0-1.7 is instruction for Mac. Please use the link to find the right steps for your operating system. 
+
+**Link:** https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository 
+
+**1.1**	Go to the main page of the repository. 
+
+**1.2** Above the list of files, click  Code.
+
+**1.3** Copy the URL for the repository.
+
+**1.4** Open Terminal.
+
+**1.5** Change the current working directory to the location where you want the cloned directory.
+
+**1.6** Type git clone, and then paste the URL you copied earlier.
+	git clone https://github.com/YOUR-USERNAME/YOUR-REPOSITORY
```

Comment:
Since this is not a live link, you should instead place it in quotes.

```suggestion
	`git clone https://github.com/YOUR-USERNAME/YOUR-REPOSITORY`
```